src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/product/show.yaml

```diff
@@ -17,6 +17,20 @@ twig_hooks:
                 template: '@SyliusAdmin/Product/Show/Content/PageBody/_taxonomy.html.twig'
             associations:
                 template: '@SyliusAdmin/Product/Show/Content/PageBody/_associations.html.twig'
+                
+        'sylius_admin.product.show.content.page_body.details':
+            simple_product:
+                template: '@SyliusAdmin/Product/Show/Content/PageBody/Details/_simpleProduct.html.twig'
+            configurable_product:
+                template: '@SyliusAdmin/Product/Show/Content/PageBody/Details/_configurableProduct.html.twig'
+                
+        'sylius_admin.product.show.content.page_body.details.simple_product':
+            shipping:
+                template: '@SyliusAdmin/Product/Show/Content/PageBody/Details/Shared/_shipping.html.twig'
+                
+        'sylius_admin.product.show.content.page_body.details.configurable_product':
+            shipping:
+                template: '@SyliusAdmin/Product/Show/Content/PageBody/Details/Shared/_shipping.html.twig'
```

Comment:
If you don't have time now skip this request, but if you can, remove the underscores from all **new templates** created inside this PR. We're changing the concept a little bit.